src/cmd/ksh93/meson.build

```diff
@@ -42,7 +42,34 @@ m_dep = cc.find_library('m', required : false)
 
 ksh93_exe = executable('ksh', ksh93_files, c_args: ksh93_c_args , include_directories: ksh93_incdir, link_with: [ libast, libcmd, libcoshell], dependencies : [ m_dep ] )
 
-# TODO: Below command run all the tests sequentially
-# We should split out the tests and check if can run them in parallel
-# This command is run from build directory
-test('ksh93 tests', ksh93_exe, args: ['../bin/run_tests.sh'])
+# TODO: Remove bin/run_tests.sh script
+#test('ksh93 tests', ksh93_exe, args: ['../bin/run_tests.sh'])
+
+test_dir = '../src/cmd/ksh93/tests/'
+
+# This variable is used by some tests while executing subtests
+shell_var = 'SHELL=' + ksh93_exe.full_path()
+
+# These are the default locales used by legacy test script
+# '' is POSIX locale
+locales = [ '', 'C.UTF-8' ]
```

Comment:
I'd prefer to be explicit and use `'POSIX'` or `'C'` but I don't know what the AST locale code does with those values (yet another reason to use the libc implementation for these types of things).

Reply:
agree, C.UTF-8 is not in glibc upstream even.. Although Debian, openSUSE, Fedora do it downstream

Reply:
For now I will keep it as it is and leave a todo comment there.